site/en/blog/storage-partitioning-deprecation-trial/index.md

```diff
@@ -0,0 +1,151 @@
+
+### DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning
+
+If you enroll in the
+`DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning`
+deprecation trial, navigating a tab to an enrolled origin will cause all
+cross-site iframes of that same origin to remain unpartitioned only for
+`Window.sessionStorage` only and only for the lifetime of that particular tab.
+Whereas the `DisableThirdPartyStoragePartitioning` deprecation trial affects all
+third-party contexts embedded within the enrolled origin, the
+`DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning`
+deprecation trial instead registers a given origin to receive unpartitioned
+access when embedded in third-party contexts.
+
+{% Aside %}
+The `DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning` trial will work via HTML `<meta>` or an `Origin-Trial` HTTP header.
+{% endAside %}
+
+Example:
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/6uxCgy48dDcfFVgkwycu.png", alt="Storage partitioning diagram after general paritoning.", width="800", height="432" %}
+
+## What does this mean for web developers?
+
+Sites should audit their usage of unpartitioned storage, service worker, and
+communication APIs in third-party contexts, and, if needed, prepare for
+third-party partitioning before these deprecation trials expire. **The intent is
+to expire these deprecation trials with M123 on March 5, 2024.**
+
+To instruct the browser to unpartition storage in third-party content embedded
+on its pages, top-level sites need to register for one or both of the
+deprecation trials and add the corresponding trial token(s) to their HTTP
+response headers (see detailed example below).
+
+Each deprecation trial is available on Windows, Mac, Linux, Chrome OS, Android,
+and Android WebView.
+
+## Participate in the Deprecation Trials
+
+The following is a brief overview of how to participate in one or both of the
+deprecation trials. For more detailed instructions, visit
+[Get started with origin trials](/docs/web-platform/origin-trials).
+
+1.  Launch Chrome M111 (or later) and ensure the
+    [`ThirdPartyStoragePartitioning`](/blog/storage-partitioning-dev-trial/)
+    flag is enabled.
+1.  Verify that the behavior of third-party content embedded in your
+    top-level site is broken by storage partitioning (if not, then no need to
+    participate in the Deprecation Trials).
+1.  Register for the deprecation trial and get a token for your domains by
+    visiting the following:
+    1.  For a top-level site to unpartition storage, service workers,
+        and communication APIs in its third-party embedded content:
+        `[DisableThirdPartyStoragePartitioning](/origintrials/#/view_trial/-8517432795264450559)`
+    1.  For a top-level site to unpartition sessionStorage across
+        navigations:
+        `[DisableThirdPartySessionStoragePartitioningAfterGeneralPartitioning](/origintrials/#/view_trial/3444127815031586817)`
```

Comment:
Same with this link rendering incorrectly

Reply:
Same here, preview issue.